### PR TITLE
Implement handling of Geometry type

### PR DESCRIPTION
**Pull request by Farami**
This allows bulk inserting and updating of NetTopologySuites Geometry type.
By default it will use SRID 4326, which is the default SRID for MSSQL.

### CODE REVIEW

**Comment by Farami** [comment]
This will resolve #88.

**Comment by tejsj** [comment]
+1

This feature would be so nice!

File: EFCore.BulkExtensions/SqlBulkOperation.cs [hunk 1]
```diff
@@ -736,6 +744,11 @@ internal static DataTable GetDataTable<T>(DbContext context, Type type, IList<T>
                         propertyType = underlyingType;
                     }
 
+                    if (propertyType == typeof(Geometry) && isSqlServer) 
+                    {
```
Comment (valdas1) [inline EFCore.BulkExtensions/SqlBulkOperation.cs:748]: Add a check for propertyType equals geometry, and change sqlServerBytesWriter (line 718) IsGeography property to false.
```
if propertyType == typeof(Geometry)
{
    sqlServerBytesWriter.IsGeography = false;
}
```

**Review by valdas1 — CHANGES_REQUESTED** [review]
Changes to EFCore.BulkExtensions/SqlBulkOperation.cs

I have a question about setting the IsGeography property of SqlServerBytesWriter to true, and the handling of the differences between geometry and geography types.

I'm not quite sure the current version would work, so if anyone could do a quick review, or write a comment, that would be great.

**Comment by borisdj** [comment]
I have merged some previous PullRequest, so could you first resolve current conflicts.
Thx for contrib.

**Comment by miloszskalecki** [comment]
Hi Guys, any news on this fix?

**Comment by borisdj** [comment]
I have managed to manually integrate the proposed solution to the latest version.

**Pull request closed without merging**